src/Adapter/Product/VirtualProductFile/Uploader/VirtualProductFileUploader.php

```diff
@@ -29,7 +29,7 @@
 namespace PrestaShop\PrestaShop\Adapter\Product\VirtualProductFile\Uploader;
 
 use ErrorException;
-use PrestaShop\PrestaShop\Adapter\Product\VirtualProductFile\Validator\VirtualProductFileValidator;
+use PrestaShop\PrestaShop\Adapter\File\Validator\VirtualProductFileValidator;
```

Comment:
Do you think think this class should be moved into the `File` namespace as well? (not blocking)

Reply:
right, we've discussed it, so if its all ok with the plan (separating files from domain) then I guess this should indeed be moved too :thinking: